### PR TITLE
add basic_OOP

### PR DESCRIPTION
**Pull request by Khiemnx**

### CODE REVIEW

File: basic_OOP/OOP.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    //----------------------------------------Homework1----------------------------------
+    abstract class Shape {
+        protected $width;
+        protected $height;
+
+        public function __construct($width, $height) {
+            $this->width = $width;
+            $this->height = $height;
+        }
+        protected function area() {
```
Comment (bintpascaliaasia) [inline basic_OOP/OOP.php]: Why don't we use abstract function?
https://www.w3schools.com/php/php_oop_classes_abstract.asp

Comment (Khiemnx) [reply basic_OOP/OOP.php]: I have changed this to abstract function
Thanks for your comment

Comment (bintpascaliaasia) [reply basic_OOP/OOP.php]: you should write like this
```
abstract protected function area();
```

Comment (Khiemnx) [reply basic_OOP/OOP.php]: yes, i have changed it
Thanks for your comment

**Pull request merged**